frontend/store/surplus.ts

```diff
@@ -0,0 +1,34 @@
+import type { SurplusAuctionFullData } from 'auctions-core/src/types';
+import { ActionContext } from 'vuex';
+import { fetchActiveSurplusAuctions } from 'auctions-core/src/surplus';
+
+interface State {
+    auctionStorage: Record<string, SurplusAuctionFullData>;
+}
+
+const getInitialState = (): State => ({
+    auctionStorage: {},
+});
+
+export const state = (): State => getInitialState();
+
+export const getters = {
+    auctionStorage(state: State) {
+        return state.auctionStorage;
+    },
+};
+
+export const mutations = {
+    addAuctionToStorage(state: State, auction: SurplusAuctionFullData) {
+        state.auctionStorage[auction.id] = auction;
```

Comment:
Adding new key to the store wouldn't reactively propagate the changes. You need to either:
- use `Vue.set` (preferred) https://github.com/sidestream-tech/unified-auctions-ui/blob/d295a3e6879b6b49348d3e5e8c84865e4444af29/frontend/store/auctions.ts#L125
- replace the whole object https://github.com/sidestream-tech/unified-auctions-ui/blob/d295a3e6879b6b49348d3e5e8c84865e4444af29/frontend/store/auctions.ts#L119-L122